pkgs/tools/misc/castty/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, libsoundio, lame }:
+
+stdenv.mkDerivation {
+  pname = "castty";
+  version = "2020-11-10";
```

```suggestion
  version = "unstable-2020-11-10";
```

Request for a release: https://github.com/dhobsd/castty/issues/41

Not a blocker.